setting-up-multifactor-authentication.md

```diff
@@ -56,6 +56,8 @@ will see a dropdown menu with three options:
 - **UI and gem signin**: UI operations and `gem signin` will require OTP code.
 - **UI and API**: UI operations, `gem signin`, `push`, `owner --add` and `owner --remove` will require OTP code.
 
+Note: If you are on the **UI only** or **UI and gem signin** authentication level, you can enable MFA on certain API keys instead of all keys defined on the **UI and API** level (see [API key scopes](http://guides.rubygems.org/api-key-scopes/#/#enable-mfa-on-specific-api-keys)).
```

Comment:
> instead of all keys defined on the **UI and API** level

This is making the sentence too long. Either we can just drop it (UI and API bullet above already explains otp requirement) or write a new sentence explaining the difference between selectively enabling mfa and otp required by default (for UI and API). Note otp required by default is not the same as enabling MFA on API key.

Reply:
Agreed, I added another sentence and tweaked the wording a bit - let me know what you think 🙂